docs/FAQ.md

```diff
@@ -33,6 +33,10 @@ revision visible again.
 
 See [revsets] and [templates] for further guidance.
 
+### How do I just see what `git log` would show me?
```

Comment:
Nit: How do I see a similar output as `git log` provides with `jj log`?

Reply:
Splitting the difference, the way I would phrase this like "How can I get `jj log` to show me what `git log` would show me?"

Reply:
Excellent. Rephrasing!